lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/util/LibertyUtils.java

```diff
@@ -148,18 +154,15 @@ public static String getRuntimeInfo(String serverXMLUri) {
             return runtime;
         }
 
-        if (findFileInWorkspace(serverXMLUri, Paths.get("WebSphereApplicationServer.properties")) != null ) {
-            runtime = "wlp";
-            libertyWorkspace.setLibertyRuntime(runtime);
-            return runtime;
-        } else if (findFileInWorkspace(serverXMLUri, Paths.get("openliberty.properties")) != null ) {
+        if (findFileInWorkspace(serverXMLUri, Paths.get("openliberty.properties")) != null) {
```

Comment:
I thought these were in the reverse order because `openliberty.properties` can exist in a WebSphere Liberty installed runtime. The presence of `WebSphereApplicationServer.properties` takes precedence.

Reply:
<img width="340" alt="Screen Shot 2022-08-26 at 3 00 42 PM" src="https://user-images.githubusercontent.com/29436451/186982290-0b56859d-2171-4dcd-b995-8865a9d797c7.png">